src/patternfly/components/Tabs/tabs.scss

```diff
@@ -0,0 +1,189 @@
+@import "../../patternfly-utilities";
+
+.pf-c-tabs-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+// Tabs
+.pf-c-tabs {
+  /* stylelint-disable */
+
+  // Primary Tab Items
+  --pf-c-tabs__item--BackgroundColor:       var(--pf-global--BackgroundColor--100);
+  --pf-c-tabs__item--current--Color:        var(--pf-global--active-color--100);
+  --pf-c-tabs__item--hover--BoxShadowColor: #8b8d8f;
+  --pf-c-tabs__item--BorderWidth:           var(--pf-global--BorderWidth--sm);
+  --pf-c-tabs__item--BorderColor:           #ededed;
+
+  // Secondary Tab Items
+  --pf-c-tabs__item--hover--Color: var(--pf-global--Color--dark-200);
+
+  // Tab Buttons
+  --pf-c-tabs__button--PaddingTop:    var(--pf-global--spacer--sm);
+  --pf-c-tabs__button--PaddingRight:  var(--pf-global--spacer--sm);
+  --pf-c-tabs__button--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-tabs__button--PaddingLeft:   var(--pf-global--spacer--sm);
+
+  // Scroll Buttons
+  --pf-c-tabs__scroll-button--Width:           var(--pf-global--spacer--xl);
+  --pf-c-tabs__scroll-button--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-tabs__scroll-button--BorderColor:     #ededed;
+
+  /* stylelint-enable */
+
+  position: relative;
+  display: inline-flex;
+
+  // Overlap secondary tab border
+  & ~ .pf-c-tabs.pf-m-tabs-secondary {
+    margin-top: -1px;
+  }
+
+  // Filled style
+  &.pf-m-fill {
+    display: flex;
+    width: 100%;
+
+    .pf-c-tabs__list,
+    .pf-c-tabs__item {
+      flex: 1;
+    }
+
+    .pf-c-tabs__item {
+      white-space: nowrap;
+
+      &:first-child {
+        border-left: none;
+      }
+
+      &:last-child {
+        border-right: none;
+      }
+
+      .pf-c-tabs__button {
+        width: 100%;
+      }
+    }
+  }
+
+  // Scroll button modifiers
+  &.pf-m-start .pf-c-tabs__scroll-button:nth-of-type(1) {
```

Comment:
I've condensed them. I think `nth-of-type` makes sense, since there should only be two scroll buttons of the class `.pf-c-tabs__scroll-button` within the direct parent container, right?